libs/application/templates/official-journal-of-iceland/src/screens/InvolvedPartyScreen.tsx

```
@@ -46,7 +49,11 @@ export const InvolvedPartyScreen = ({
           involvedParty.id,
         )
 
-        updateApplication(updatedAnswers)
+        updateApplication(updatedAnswers, () => {
+          submitApplication(DefaultEvents.SUBMIT, () => {
+            refetch && refetch()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)